script.js

```diff
@@ -0,0 +1,12 @@
+
+<script>
+    function calc() {
+        var x = Number (document.getElementById)("first").value;
```

Comment:
You've got a typo here causing this to throw an error when attempting to run it. `document.getElementById` gets the element, then `.value` gets the number typed in as a string. The `Number()` has to surround the whole thing. 

```JavaScript
var x = Number(document.getElementById("first").value)
```

What you've written is attempting to turn `document.getElementById` into a number and since it is a function, you can't. Code is all about the tiny details, this simple `)` being in the wrong place breaks everything. Definitely spend time with the TAs in the future if you're having trouble getting this to work correctly, they can help spot things like this so you don't lose points and more importantly you start spotting them yourself with that practice.